api/docker-compose.deploy.yml

```diff
@@ -40,18 +40,18 @@ services:
       - "80:80"
     labels:
       - "traefik.enable=true"
-      - "traefik.http.routers.whoami.rule=Host(`prism-api.ovio.org`)"
+      - "traefik.http.routers.whoami.rule=Host(${HOSTNAME:?'Must provide hostname'})"
```

Comment:
Is this properly getting set on the new instance? Anyway to confirm?

Reply:
I think this is missing the ` ` no?

Reply:
Fixed ✅